.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
yep, exactly. mock-fs is only compatible with <=6, so if you cd into the dir and type `nvm use` it will grab a compatible node version